concepts/05 Widgets/Scheduler/00 Getting Started with Scheduler/10 Configure Views.md

```diff
@@ -0,0 +1,149 @@
+The **Scheduler** supports the following views:
+
+* [Day](/Documentation/Guide/Widgets/Scheduler/Views/View_Types/#Day_View) (default)
+
+* [Week and WorkWeek](/Documentation/Guide/Widgets/Scheduler/Views/View_Types/#Week_and_WorkWeek_Views) (week is added by default)
+
+* [Month](/Documentation/Guide/Widgets/Scheduler/Views/View_Types/#Month_View)
+
+* [Timeline](/Documentation/Guide/Widgets/Scheduler/Views/View_Types/#Timeline_Views)
+
+* [Agenda](/Documentation/Guide/Widgets/Scheduler/Views/View_Types/#Agenda_View)
+
+* [Customized views](/Documentation/Guide/Widgets/Scheduler/Views/Customize_Individual_Views/).
+
+The default views are *"day"* and *"week"*. To use custom ones, specify the [views[]](/Documentation/ApiReference/UI_Widgets/dxScheduler/Configuration/views/) property. In this tutorial, we will use *"day"*, *"week"*, *"month"*, and *"timelineWeek"* views.
+
+The default view is *"day"*. You can change it in the [currentView](/Documentation/ApiReference/UI_Widgets/dxScheduler/Configuration/#currentView) property - we will set it to *"week"*.
+
+You can customize any view. For this, add an object to **views** and specify its [type](/Documentation/ApiReference/UI_Widgets/dxScheduler/Configuration/views/#type) and other properties. In this tutorial, we will set the first and last hours of the *"week"* view's time scale to 10 and 22. For this, we will use the [startDayHour](/Documentation/ApiReference/UI_Widgets/dxScheduler/Configuration/views/#startDayHour) and [endDayHour](/Documentation/ApiReference/UI_Widgets/dxScheduler/Configuration/views/#endDayHour) properties. 
```

Comment:
Rewrite. Start with the `views` array, what it can contain, what are the defaults. Then, move on to how it is used in this tutorial. Finish with setting the `currentView`.